server/users.ts

```diff
@@ -1232,11 +1232,6 @@ export class User extends Chat.MessageContext {
 		force = false
```

Comment:
Can we also remove this parameter?

Reply:
oh yeah, looks like it

half surprised the linter isn't complaining, honestly.

Reply:
The linter allows unused parameters by default because it's sometimes necessary for compatibility reasons. There's probably a setting to change it.